languageserver/go.mod

```diff
@@ -177,3 +186,7 @@ require (
 	modernc.org/memory v1.5.0 // indirect
 	modernc.org/sqlite v1.21.1 // indirect
 )
+
+replace github.com/onflow/cadence-tools/test => ../test
+
+replace github.com/onflow/cadence => ../../cadence
```

Comment:
`replace` statements should not be comitted

```suggestion
```

Reply:
maybe refer to the commit, until we tag a release

Reply:
I think I have tried that before, but it didn't work with forks. I will check again though :+1: